app/reducer.js

```diff
@@ -31,7 +31,7 @@ function trackNewSubject(state, action) {
 function prependTweets(state, action) {
   const subjects = mapState(state, action, s => {
     const tweets = action.tweets.map(t =>
-      Map(t).merge({ read: false })
+      Map(t).set('read', false)
```

Comment:
Good call here, but your suggestion to keep two lists instead of one was even better 👍